### DISCUSSION TITLE
Add Task Title Description and Priority

### DISCUSSION THREAD

**Discussion by workgroupengineering** [VsixCommunity/Community.VisualStudio.Toolkit#501] [Q&A] [answered]
Hi,

I would like to run some tasks and set the title, description and priority as in the following image.
How can I do?

![immagine](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/assets/12531229/27f651c2-1e8e-4714-a07d-35b79d886858)

**Comment by jamesc-skyward** [comment, marked as answer]
Is what you're asking for the same thing as this? https://www.vsixcookbook.com/recipes/showing-progress.html#task-status-center
